modules/consensus/consensusset.go

```diff
@@ -222,6 +228,12 @@ func (cs *ConsensusSet) CurrentBlock() (block types.Block) {
 	return block
 }
 
+// Flush will block until the consensus set has finished all in-progress
+// routines.
+func (cs *ConsensusSet) Flush() error {
+	return cs.tg.Flush()
+}
```

Comment:
*It is my plan to expose a `Flush()` function on all modules, to accompany the `Close()` function which already exists in all modules.